packages/tooltip/src/Tooltip/Tooltip.tsx

```diff
@@ -155,6 +155,12 @@ function Tooltip({
             onBlur: (e: MouseEvent) => {
               userTriggerHandler('onBlur', e);
               handleClose();
+
+              // If a user clicks on the tooltip trigger and immediately navigates to a new window/tab and then back, document.activeElement is still the trigger which will cause the
+              // tooltip to reappear even if not hovered. This blurs the last active element to prevent it from unexpectedly reappearing.
+              if (document.activeElement instanceof HTMLElement) {
```

Comment:
Can you add a test to ensure that after tabbing to the next focusable item, the focus is on the correct element?

Reply:
This will actually move focus to the `body` element. I wonder if that is acceptable or if we'd rather not address this issue altogether...

| pre-change | post-change |
| - | - |
| 1. click trigger: trigger focused<br/>2. navigate to new browser tab/window and back<br/>3. trigger still focused: tooltip reappears | 1. click trigger: trigger focused<br/>2. navigate to new browser tab/window and back<br/>3. body focused: tooltip does not appear<br/>4. hit tab and trigger is focused|